packages/toolkit/src/view/settings/user/UserProfileTab.tsx

```diff
@@ -441,6 +468,34 @@ export const UserProfileTab = () => {
           </div>
         </form>
       </Form.Root>
+      <Dialog.Root open={isOpenProfileAvatar}>
+        <Dialog.Content className="w-[400px]">
+          <Dialog.Header>
+            <Dialog.Title>Crop your new profile picture</Dialog.Title>
+          </Dialog.Header>
+          <div className="flex items-center justify-center">
+            <AvatarEditor
+              ref={editorRef}
+              image={profileAvatar}
+              width={150}
+              height={150}
+              border={50}
+              color={[0, 0, 0, 0.8]}
+              scale={1}
```

Comment:
I think the cropped area is very hard to see, is there any other way to make it more obvious?